src/stylesheets/components/_range-input.scss

```diff
@@ -0,0 +1,21 @@
+.usa-range{
+    &::-webkit-slider-runnable-track {
+      background: #c9c9c9;
+      border: none;
+    }
+  
+    &::-webkit-slider-thumb {
+      background-color: #5994f6;
+      border: 1px solid #5994f6;
+      box-shadow: none;
+    }
+  
+    &:focus{
+      &::-webkit-slider-thumb {
+        border: 1px solid #2e2e2a;
+        background-color: #81aefc;
+      }
+    }
+  }
+  
```

Comment:
Let's use the variable [theme tokens](https://federalist-0ad5a602-ca98-4a7e-8d6e-d9ece7bc4cf8.app.cloud.gov/site/gsa/sam-styles/components/detail/theme-color.html) instead of hard coding hex values

Reply:
Updated.

Reply:
@cwolf10 Might have not been pushed

Reply:
doh. should be pushed now